pos_mail/__manifest__.py

```diff
@@ -26,6 +26,7 @@
         "views/views.xml",
     ],
     "demo": [
```

Comment:
ok. I meant do not attach the template as demo. It should be a data, so in the DB without demo it is presented to.